README.md

```diff
@@ -1,76 +1,41 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# AwesomeBooks
 
-> One paragraph statement about the project.
+This is a wesbite where you can add and remove books.
 
-![screenshot](./app_screenshot.png)
+![screenshot](./assets/screenshots/screenshot.PNG)
 
-Additional description about the project and its features.
+This project is my portfolio.
 
 ## Built With
+- Javascript
+- HTML
+- CSS
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo](https://mugishasam123.github.io/awesomebooks/)
 
 ## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
+To get the content of this project locally you need to run this command in your terminal:
+- git clone your https://github.com/matovu-farid/My-Portfolio.git
+- cd project My-Portfolio
```

Comment:
- [ ] I would recommend mentioning all the steps to run the project locally. Here next step is `open index.html` file to see the app running.